pages/a/monitor-your-cratedb-cluster.html

```diff
@@ -0,0 +1,107 @@
+title: Monitoring a scalable SQL DB
+link: https://crate.io/blog/monitoring-a-scalable-sql-db
+author: Claus Matzinger
+description: What metrics to look for in a scalable SQL DB.
+created: 2016-11-09
+post_name: monitoring-a-scalable-sql-db
+status: publish
+post_type: post
+tags: developernews, features
+category: post
+
+
+
+CrateDB is running in production in companies around the world, with users applying a range of different monitoring solutions to ensure cluster health and responsiveness. We hope that this blogpost can help provide an overview on how and what we recommend monitoring when running CrateDB. Effective monitoring should allow our users to identify potential hardware/software issues or to see if changes to cluster size would benefit performance.
```

Comment:
Instead of 

> We hope that this blogpost can help provide an overview on how and what we recommend monitoring when running CrateDB.

I'd write something like:

>This blogpost provides an overview...